vendor/installers/windows/INSTALL_COSMOS.bat

```diff
@@ -166,6 +161,8 @@ if !ARCHITECTURE!==x86 (
   ) else (
     @echo Successfully installed 32-bit Ruby >> !COSMOS_INSTALL!\INSTALL.log
   )
+  call !COSMOS_INSTALL!\Vendor\Ruby\bin\ridk install 1 2 3
```

Comment:
What does this do? ridk install 1 2 3

Reply:
Thats the new way to install devkit (msys64).   

Also devkitvars.bat has been replaced by "ridk enable"